src/pages/Graph/GraphToolbar/GraphFind.tsx

```diff
@@ -698,6 +701,13 @@ export class GraphFind extends React.Component<GraphFindProps, GraphFindState> {
       case 'op':
       case 'operation':
         return { target: 'node', selector: `[${CyNode.aggregateValue} ${op} "${val}"]` };
+      case 'rank': {
+        if (!this.props.rank || this.props.rankBy.length === 0) {
+          return undefined;
+        }
+        const s = this.getNumericSelector(CyNode.rank, op, val, expression, isFind);
+        return s ? { target: 'node', selector: s } : undefined;
+      }
```

Comment:
We have a convention (hidden, yes, known probably only be me, but...) to enable options that are required for the use of an operand used in find or hide.  For example, see `mtls`.  If you use this in an expression we automatically ensure that the `Security` option is enabled.  Try it, I think we want to do the same here.

Also, I think we should generate an error if the `val` is out of range.

Reply:
This helped in discovering a subtle bug where node scoring was not zeroing out old scores/ranks correctly when no selection criteria was passed in.

I added in the automatically enable feature. It's a nice touch but just for ranking, users will still need to select their scoring criteria e.g. you can have rank checked but not have any criteria such as "inbound edges" checked.

Reply:
Nice, glad it surfaced an issue.  It's unlikely that users will use Find/Hide withhout having set any rank criteria, so just ensuring it's enabled should be sufficient.